### ISSUE TITLE
Uncaught TypeError: Cannot read property 'getChildValueConstraints' of null

### ISSUE THREAD

**Issue by shlomiassaf** [mulesoft/api-workbench#35]
[Enter steps to reproduce below:]

1. Create a Test.raml file, this file represents a DateType
2. Paste the first line: __#%RAML 1.0 DataType__
3. Press enter, get the error, try to add any property (e.g: type: object) raises the error.

It should be noted that once the DataType is written (correctly) the parent including it does not complain and the outline shows the schema correctly.

Error reproduced in Win8.1 & OSX (El Capitan)
**Atom Version**: 1.1.0
**System**: Microsoft Windows 8.1 Pro
**Thrown From**: [api-workbench](https://github.com/mulesoft/api-workbench) package, v0.8.6

### Stack Trace

Uncaught TypeError: Cannot read property 'getChildValueConstraints' of null

```
At C:\Users\Shlomi.ASSAF\.atom\packages\api-workbench\main.js:9

TypeError: Cannot read property 'getChildValueConstraints' of null
  at C:\Users\Shlomi.ASSAF\.atom\packages\api-workbench\main.js:9:27944
  at C:\Users\Shlomi.ASSAF\.atom\packages\api-workbench\main.js:29:6733
  at Array.forEach (native)
  at k (C:\Users\Shlomi.ASSAF\.atom\packages\api-workbench\main.js:29:6595)
  at h (C:\Users\Shlomi.ASSAF\.atom\packages\api-workbench\main.js:29:2884)
  at Object.m [as getSuggestions] (C:\Users\Shlomi.ASSAF\.atom\packages\api-workbench\main.js:28:31236)
  at C:\Users\Shlomi.ASSAF\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\autocomplete-plus\lib\autocomplete-manager.js:286:56
  at Array.forEach (native)
  at AutocompleteManager.module.exports.AutocompleteManager.getSuggestionsFromProviders (C:\Users\Shlomi.ASSAF\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\autocomplete-plus\lib\autocomplete-manager.js:265:17)
  at AutocompleteManager.getSuggestionsFromProviders (C:\Users\Shlomi.ASSAF\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\autocomplete-plus\lib\autocomplete-manager.js:3:61)
  at AutocompleteManager.module.exports.AutocompleteManager.findSuggestions (C:\Users\Shlomi.ASSAF\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\autocomplete-plus\lib\autocomplete-manager.js:252:19)
  at C:\Users\Shlomi.ASSAF\AppData\Local\atom\app-1.1.0\resources\app.asar\node_modules\autocomplete-plus\lib\autocomplete-manager.js:3:61

```

### Commands

```
     -0:57.9.0 autocomplete-plus:activate (atom-text-editor.editor.is-focused)
     -0:37.2.0 project-find:show-in-current-directory (span.name.icon.icon-file-directory)
     -0:34.2.0 tree-view:add-file (span.name.icon.icon-file-directory)
  5x -0:30.7.0 core:backspace (atom-text-editor.editor.mini.is-focused)
     -0:26 core:confirm (atom-text-editor.editor.mini.is-focused)
     -0:22.9.0 core:select-all (atom-text-editor.editor.is-focused)
  2x -0:22.7.0 core:copy (atom-text-editor.editor.is-focused)
     -0:03.8.0 core:paste (atom-text-editor.editor.is-focused)
     -0:03.2.0 editor:newline (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "themes": [
      "atom-light-ui",
      "atom-dark-syntax"
    ]
  },
  "api-workbench": {}
}
```

### Installed Packages

```coffee
# User
.bin, vundefined
api-workbench, v0.8.6
atom-typescript, v7.0.1
language-elm, v1.0.3
linter, v1.6.0

# Dev
No dev packages
```

**Comment by sichvoge** [comment]
I can confirm that this happens to me as well. We will investigate. Thanks for raising. In the meanwhile, you could create a library of data types instead of using fragments where you would define each type separately.

**Comment by shlomiassaf** [comment]
Thank you for the update.

I prefer separating types as much as possible otherwise maintenance get hard.

For the time being, I build the type in the main file and copy paste afterwards :)

**Comment by sichvoge** [comment]
You can separate your types from your main file using the library fragments. You don't need to put everything in a single file yet.

**Comment by petrochenko-pavel-a** [comment]
Fix will be published in 0.8.7
